lib/mangadex_import.rb

```diff
@@ -0,0 +1,105 @@
+class MangadexImport
+  attr_reader :file_location
+
+  LANGUAGES = {
+    'arabic' => 'ar',
+    'bulgarian' => 'bg',
+    'burmese' => 'my',
+    'catalan' => 'ca',
+    'chinese (trad)' => 'zh_Hant',
+    'chinese (simp)' => 'zh_Hans',
+    'english' => 'en',
```

Comment:
should this also be `en_jp` because when saving with Chapter it will fail validation...